### ISSUE TITLE
connecting to SQL server

### ISSUE THREAD

**Issue by Lincoln-Hannah** [TidierOrg/TidierDB.jl#58]
Could you add a documentation example of connecting to MS SQL Server ?

**Comment by drizk1** [comment]
Hmm, thats strange, normally it should be nearly immediate. 

I was just able to successfully do it. 
```
using Pkg; Pkg.add(name = "TidierDB", version = "0.3.2")
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
    Updating `~/.julia/environments/v1.10/Project.toml`
  [86993f9b] ~ TidierDB v0.3.2 `https://github.com/TidierOrg/TidierDB.jl#main` ⇒ v0.3.2
    Updating `~/.julia/environments/v1.10/Manifest.toml`
  [86993f9b] ~ TidierDB v0.3.2 `https://github.com/TidierOrg/TidierDB.jl#main` ⇒ v0.3.2

(@v1.10) pkg> status TidierDB
Status `~/.julia/environments/v1.10/Project.toml`
  [86993f9b] TidierDB v0.3.2
```

Another option would be the following (which will be the latest version as well.
```
using Pkg; Pkg.add(url = "https://github.com/TidierOrg/TidierDB.jl" )
```

**Comment by Lincoln-Hannah** [comment]
i think its a fire-wall blocking the package on my end. 
I'll keep trying. Many thanks.

**Comment by drizk1** [comment]
Sure thing. 

One other option if older versions are available would be to try v0.2.4 and then library in ODBC, and use `ODBC.Connection` to establish a connection. 

From there you could `set_sql_mode(:mssql)` ( note the symbol is because it's an older version before the switch in v0.3.0) and that should work
